src/js/classes/ArrayValidator.ts

```diff
@@ -33,10 +35,10 @@ class ArrayValidator<T extends BaseValidator<any>> extends BaseValidator<ValueOf
      * @param {Dependent} [dependent]
      */
     constructor(of: T,
-                          defaultMessage: string = 'This field has to be an array',
-                          mutationFunc?: MutationFunction,
-                          onError?: ErrorFunction,
-                          dependent?: Dependent) {
+                defaultMessage?: string,
```

Comment:
This means that the error message for "hey, this isn't an array!" will be more generic going forward, correct?

Reply:
Not entirely. If you look at the validate function, you will see that if there is a t function given to it (which is the case in almost all situations that use formifly as intended) it will return a specific error message if the given value is no array